plugins/htpasswd/.editorconfig

```diff
@@ -0,0 +1,12 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# 2 space indentation
+[{.,}*.{js,yml,yaml}]
```

Comment:
I'd add here `ts`

Reply:
You are right with the config change, but I think this change should be done in other moment, to migrate as close as the original repo is

Reply:
agree